chef/cookbooks/crowbar/templates/default/crowbar-webserver.pill.erb

```diff
@@ -1,7 +1,7 @@
 Bluepill.application("crowbar-webserver") do |app|
```

Comment:
Maybe not as part of this PR but as a followup, let's drop all that bluepill crap.